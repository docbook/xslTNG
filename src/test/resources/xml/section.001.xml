<?xml version="1.0" encoding="utf-8"?>
<article xmlns="http://docbook.org/ns/docbook" version="5.0" role="landscape">
   <title>Article wrapper</title>
   <para>See <xref linkend="c2s1s1s2"/>.</para>
   <section xml:id="c2s1">
      <title>Section 1</title>
      <para>blah.</para>
      <section xml:id="c2s1s1">
         <title>Section 1.1</title>
         <para>blah.</para>
         <section xml:id="c2s1s1s1">
            <title>Section 1.1.1</title>
            <para>blah.</para>
<table xmlns="http://docbook.org/ns/docbook" version="5.0" xml:id="d1e3" frame="all" orient="port">
      <info>
        <title>Table title</title>
      </info>
      <tgroup cols="4" colsep="1" rowsep="1">
        <colspec colname="b" colnum="2"/>
        <colspec colname="c" colnum="3"/>
        <colspec colname="d"/>
        <spanspec spanname="cd" namest="c" nameend="d"/>
        <tbody>
          <row>
            <entry>a1</entry>
            <entry morerows="1" namest="b" nameend="c">b1-c2</entry>
            <entry>d1</entry>
          </row>
          <row>
            <entry morerows="2">a2-a4</entry>
            <!-- b2, c2 spans from above -->
            <entry>d2</entry>
          </row>
          <row>
            <!-- a3 spans from above -->
            <entry>b3</entry>
            <entry>c3</entry>
            <entry>d3</entry>
          </row>
          <row>
            <!-- a4 spans from above -->
            <entry>b4</entry>
            <entry spanname="cd">c4-d4</entry>
          </row>
          <row>
            <entry>a5</entry>
            <entry>b5</entry>
            <!-- c4 is missing -->
            <entry colname="d">d5</entry>
          </row>
        </tbody>
      </tgroup>
    </table>
            <section xml:id="c2s1s1s1s1">
               <title>Section 1.1.1.1</title>
               <para>blah.</para>
               <section xml:id="c2s1s1s1s1s1">
                  <title>Section 1.1.1.1.1</title>
                  <para>blah.</para>
               </section>
               <section xml:id="c2s1s1s1s1s2">
                  <title>Section 1.1.1.1.2</title>
                  <para>blah.</para>
               </section>
               <section xml:id="c2s1s1s1s1s3">
                  <title>Section 1.1.1.1.3</title>
                  <para>blah.</para>
               </section>
            </section>
            <section xml:id="c2s1s1s1s2">
               <title>Section 1.1.1.2</title>
               <para>blah.</para>
            </section>
            <section xml:id="c2s1s1s1s3">
               <title>Section 1.1.1.3</title>
               <para>blah.</para>
            </section>
         </section>
         <section xml:id="c2s1s1s2">
            <title>Section 1.1.2</title>
            <para>blah.</para>
         </section>
         <section xml:id="c2s1s1s3">
            <title>Section 1.1.3</title>
            <para>blah.</para>
         </section>
      </section>
      <section xml:id="c2s1s2">
         <title>Section 1.2</title>
         <para>blah.</para>
      </section>
      <section xml:id="c2s1s3">
         <title>Section 1.3</title>
         <para>blah.</para>
      </section>
   </section>
</article>
