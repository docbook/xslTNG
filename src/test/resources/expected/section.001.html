<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><title>Article wrapper</title><meta name="viewport" content="width=device-width, initial-scale=1.0"/><link href="https://purl.org/dc/elements/1.1/" rel="schema.dc"/><meta content="2011-04-22T17:02:00-06:00" name="dc.modified"/><meta content="DocBook xslTNG" name="generator"/><link href="./css/docbook.css" rel="stylesheet" media="screen"/></head><body class="home"><nav class="top"></nav><main><article class="article component landscape"><header><h1>Article wrapper</h1></header><div class="list-of-titles"><div class="lot toc"><div class="title">Table of Contents</div><ul class="toc"><li><a href="#c2s1"><span class="label">1</span><span class="sep">. </span>Section 1</a><ul class="toc"><li><a href="#c2s1s1"><span class="label">1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1</a><ul class="toc"><li><a href="#c2s1s1s1"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1.1</a><ul class="toc"><li><a href="#c2s1s1s1s1"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1.1.1</a><ul class="toc"><li><a href="#c2s1s1s1s1s1"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1.1.1.1</a></li><li><a href="#c2s1s1s1s1s2"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.1.1.1.2</a></li><li><a href="#c2s1s1s1s1s3"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.1.1.1.3</a></li></ul></li><li><a href="#c2s1s1s1s2"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.1.1.2</a></li><li><a href="#c2s1s1s1s3"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.1.1.3</a></li></ul></li><li><a href="#c2s1s1s2"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.1.2</a></li><li><a href="#c2s1s1s3"><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.1.3</a></li></ul></li><li><a href="#c2s1s2"><span class="label">1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.2</a></li><li><a href="#c2s1s3"><span class="label">1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.3</a></li></ul></li></ul></div></div><div class="db-bfs"><p>See <a href="#c2s1s1s2" class="xref xref-section">Section <span class="label">1<span class="sep">.</span>1<span class="sep">.</span>2</span>, “Section 1.1.2”</a>.</p></div><section id="c2s1" class="section"><header><h2><span class="label">1</span><span class="sep">. </span>Section 1</h2></header><p>blah.</p><section id="c2s1s1" class="section"><header><h3><span class="label">1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1</h3></header><p>blah.</p><section id="c2s1s1s1" class="section"><header><h4><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1.1</h4></header><p>blah.</p><div id="d1e3" class="formalobject table"><header><div class="title">Table <span class="label">1</span><span class="sep">. </span>Table title</div></header><table><tbody><tr><td class="bleft btop colsep rowsep">a1</td><td class="btop colsep rowsep" colspan="2" rowspan="2">b1-c2</td><td class="btop colsep rowsep">d1</td></tr><tr><td class="bleft colsep rowsep" rowspan="3">a2-a4</td><td class="colsep rowsep">d2</td></tr><tr><td class="colsep rowsep">b3</td><td class="colsep rowsep">c3</td><td class="colsep rowsep">d3</td></tr><tr><td class="colsep rowsep">b4</td><td class="colsep rowsep" colspan="2">c4-d4</td></tr><tr><td class="bleft colsep rowsep">a5</td><td class="colsep rowsep">b5</td><td class="colsep empty rowsep"></td><td class="colsep rowsep">d5</td></tr></tbody></table></div><section id="c2s1s1s1s1" class="section"><header><h5><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1.1.1</h5></header><p>blah.</p><section id="c2s1s1s1s1s1" class="section"><header><h5><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1</span><span class="sep">. </span>Section 1.1.1.1.1</h5></header><p>blah.</p></section><section id="c2s1s1s1s1s2" class="section"><header><h5><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.1.1.1.2</h5></header><p>blah.</p></section><section id="c2s1s1s1s1s3" class="section"><header><h5><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.1.1.1.3</h5></header><p>blah.</p></section></section><section id="c2s1s1s1s2" class="section"><header><h5><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.1.1.2</h5></header><p>blah.</p></section><section id="c2s1s1s1s3" class="section"><header><h5><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.1.1.3</h5></header><p>blah.</p></section></section><section id="c2s1s1s2" class="section"><header><h4><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.1.2</h4></header><p>blah.</p></section><section id="c2s1s1s3" class="section"><header><h4><span class="label">1<span class="sep">.</span>1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.1.3</h4></header><p>blah.</p></section></section><section id="c2s1s2" class="section"><header><h3><span class="label">1<span class="sep">.</span>2</span><span class="sep">. </span>Section 1.2</h3></header><p>blah.</p></section><section id="c2s1s3" class="section"><header><h3><span class="label">1<span class="sep">.</span>3</span><span class="sep">. </span>Section 1.3</h3></header><p>blah.</p></section></section></article></main><nav class="bottom"></nav></body></html>